src/actions/index.js

```diff
@@ -1,17 +1,12 @@
-import content from '../apis/content';
-import { FETCH_EVENTS, DELETE_EVENT, MODAL_OPEN, MODAL_CLOSE } from './actionTypes';
+import { MODAL_OPEN, MODAL_CLOSE, GET_PAGES } from './actionTypes';
 
-//Event Actions Examples
-export const fetchEvents = () => async (dispatch) => {
-  const res = await content.get('/events');
-
-  dispatch({ type: FETCH_EVENTS, payload: res.data });
-};
-
-export const deleteEvent = (id) => async (dispatch) => {
-  // const res = await content.delete(`/events/${id}`);
+import getAPIRoot from '../apis/content';
+import axios from 'axios';
 
-  dispatch({ type: DELETE_EVENT, payload: id });
+export const getPages = () => async (dispatch) => {
+  const response = await axios.get(`${getAPIRoot}/wp/v2/pages?page=1&per_page=100`);
```

Comment:
Oh we need logic to handle if there is more that 100 pages

Reply:
This call is just getting the static content, which is unlikely to exceed 100 pages. 

Logic should be included in the calls for dynamic content such as post, events, etc since those

Reply:
it is **unlikey** but we shouldnt rule it out